app/controllers/shopify_app/sessions_controller.rb

```diff
@@ -1,5 +1,5 @@
 module ShopifyApp
-  class SessionsController < ApplicationController
+  class SessionsController < ActionController::Base
     include ShopifyApp::SessionsConcern
```

Comment:
yeah I get tripped up by that too. Every time I always wonder if anyone is actually using this concern in their own controller or if we could simplify it and move this code into the controller.